maker-frontend/src/MakerApp.tsx

```diff
@@ -137,12 +154,13 @@ export default function App() {
 
                         <GridItem colSpan={2} textAlign="center">
                             <Button
-                                disabled={isCreatingNewCfdOrder || orderPrice === "0"}
+                                disabled={isCreatingNewCfdOrder || shortPrice === "0"}
                                 variant={"solid"}
                                 colorScheme={"blue"}
                                 onClick={() => {
                                     let payload: CfdSellOrderPayload = {
-                                        price: Number.parseFloat(orderPrice),
+                                        price: Number.parseFloat(shortPrice),
+                                        price_long: Number.parseFloat(longPrice),
                                         min_quantity: Number.parseFloat(minQuantity),
                                         max_quantity: Number.parseFloat(maxQuantity),
                                         // TODO: Populate funding rate from the UI
```

Comment:
Second part of the ticket is yet to be solved
("change the button name from 'create/update sell order' to 'create/update offer'")

it's a few lines below this comment, on line 174 (can't suggest changes somehow in the github UI....)